enterprise/admin/forms.py

```diff
@@ -732,3 +734,8 @@ def clean(self):
 
             raise ValidationError(message)
         return cleaned_data
+
+
+@admin.register(BulkCatalogQueryUpdateCommandConfiguration)
+class BulkCatalogQueryUpdateCommandConfigurationAdmin(ConfigurationModelAdmin):
+    pass
```

Comment:
does just having a pass here work? We don't need to even have a `model = BulkCatalogQueryUpdateCommandConfiguration`?

Reply:
Testing locally, it seems ugly but working.